packages/visx-xychart/test/components/LineSeries.test.tsx

```diff
@@ -85,6 +85,21 @@ describe('<LineSeries />', () => {
       { showTooltip, hideTooltip },
     );
   });
+
+  it('should use colorAccessor if passed', () => {
+    const {container} = render(
+      <DataContext.Provider value={getDataContext(series)}>
+        <svg>
+          <LineSeries
+            dataKey={series.key}
```

Comment:
looks like linting is complaining that this isn't a single line

```tsx
<LineSeries dataKey={series.key}·{...series}·colorAccessor={(_)·=>·'banana'} />
```

You could also run `yarn format` in the monorepo root, but sometimes that will update other files unintentionally (you can skip commit-ing them)

Reply:
yarn format run. Looks like it picked up a `,`->`;` in BaseLineSeries as well